core/docs/spark-profiling-tool.md

```diff
@@ -580,6 +499,90 @@ The _Auto-Tuner_ output has 2 main sections:
   - 'spark.sql.adaptive.enabled' should be enabled for better performance.
   ```
 
+### Generating Visualizations
+
+- Print SQL Plans (--print-plans option):
+Prints the SQL plan as a text string to a file named `planDescriptions.log`.
+For example if your application id is app-20210507103057-0000, then the
+filename will be `planDescriptions.log`
```

Comment:
This is kind of an odd example?   Maybe remove it.  I think the file is always called just `planDescriptions.log`?

Reply:
Agreed.  I'll remove it.